### PR TITLE
Wdi10 update

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -10,248 +8,408 @@ See also: https://github.com/ga-wdi-lessons/ruby-oop
...
-    else
-      @hunger_level = new_hunger_level
+<details>
+  <summary><strong>How can we set up our Person class so that it has <code>name</code> and <code>age</code> attributes. We should be able to read and modify <code>name</code>, but only read <code>age</code>.</strong></summary>
+
+  ```rb
+  class Person
+    # This allows us to both read and write to the name and hunger_level attributes
+    attr_accessor :name
+    attr_reader :age
```
Comment (jshawl) [inline readme.md]: recommend introducing these separately.

e.g. start with:
1. getter and setter methods
2. attr_reader & writer
3. attr_accessor

File: readme.md [hunk 2]
```diff
@@ -10,248 +8,408 @@ See also: https://github.com/ga-wdi-lessons/ruby-oop
...
+  - Start with an `@` (e.g., `@name`).
+  - Used to store attributes for a given instance.
+  - Can be used in any instance method in the class.
+  - Very common in ruby.
+
+  #### Class
+
+  - Start with 2 `@`s (e.g., `@@person_count`).
+  - One copy shared between all instances.
+  - Not very common. Over-use is considered bad practice.
```
Comment (jshawl) [inline readme.md:90]: re: your question about why use class vars - nearly every example i've seen involves counting the instances created from the class e.g. `@@num_people`

File: readme.md [hunk 3]
```diff
@@ -10,248 +8,408 @@ See also: https://github.com/ga-wdi-lessons/ruby-oop
...
-    # SETTERS
-    @hunger_level = -10
-    hunger_level = -10
-    self.hunger_level = -10
+  adrian = Person.new("Adrian", 29)
+  adrian.say_name
+  # => "Hi, my name is Adrian."
+  ```
+
+  > We could also use `self.name` in place of the instance variable `@name`. This is actually preferable. We'll explain why later...
```
Comment (jshawl) [inline readme.md]: http://stackoverflow.com/questions/1693243/instance-variable-self-vs `self.` is more reliable

File: readme.md [hunk 4]
```diff
@@ -10,248 +8,408 @@ See also: https://github.com/ga-wdi-lessons/ruby-oop
...
+      # Whenever we create a new person, we push it into the `@@people` array
+      @@people.push(self)
+    end
+
+    def say_name
+      puts "Hi, my name is " + @name + "."
+    end
+
+    # Here we're creating a method that retrieves the value of `@@people`
+    def Person.get_people
```
Comment (jshawl) [inline readme.md]: `def self.get_people`, right?

File: readme.md [hunk 5]
```diff
@@ -10,248 +8,408 @@ See also: https://github.com/ga-wdi-lessons/ruby-oop
...
 
+<!-- AM: I'm going to add to this exercise. -->
+
+Clone down [this repo](https://github.com/ga-wdi-exercises/superheros) and follow the instructions in the Readme.
 
 
 ## Visibility
 
+<!-- AM: The placement of this seems kind of weird. Is this covered enough in the previous lesson? Or should I integrate it into the earlier review section? -->
+
```
Comment (jshawl) [inline readme.md:447]: this is gonna go right over most people's heads. would rename `Visibility` to `Bonus`

File: readme.md (hunk 4 above)
Comment (amaseda) [inline readme.md]: Was hoping to replace with `self` later when refactoring but yeah, I should just introduce it here.

File: readme.md (hunk 5 above)
Comment (amaseda) [inline readme.md:447]: :+1:

**Pull request merged**